linera-service/src/benchmark.rs

```diff
@@ -0,0 +1,266 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use clap::Parser as _;
+use fungible::{Account, AccountOwner, FungibleTokenAbi, InitialState, Parameters};
+use futures::future::join_all;
+use linera_base::{
+    async_graphql::InputType,
+    data_types::Amount,
+    identifiers::{ApplicationId, ChainId, Owner},
+};
+use linera_execution::system::SystemChannel;
+use linera_service::cli_wrappers::{ApplicationWrapper, ClientWrapper, Network};
+use port_selector::random_free_tcp_port;
+use rand::{Rng as _, SeedableRng};
+use serde_json::Value;
+use std::{collections::BTreeMap, path::Path, sync::Arc, time::Duration};
+use tempfile::tempdir;
+use tokio::time::Instant;
+
+#[derive(clap::Parser)]
+#[command(
+    name = "linera-benchmark",
+    version = clap::crate_version!(),
+    about = "Run benchmarks against a Linera network",
+)]
+enum Args {
+    Fungible {
```

Comment:
Is this a subcommand?

Reply:
I wonder how much work it would be to move the (low-level native transfer) benchmark command of `linera` here.

Reply:
And then we could also remove the `benchmark` feature, I think?
Anyway, I'd prefer to do that in a separate PR.

Reply:
sounds good!